transformer/external/starlarktransformer.go

```diff
@@ -68,6 +68,13 @@ const (
 	fswriteFnName                = "write"
 	fspathBaseFnName             = "pathbase"
 	fspathRelFnName              = "pathrel"
+
+	// encryption functions
+	encaescbcpbkdfFnName = "encaescbcpbkdf"
```

Comment:
I think we have to reevaluate starlark function naming conventions. These function names are almost unreadable. Can you please check the conventions and come with with alternatives

Reply:
Shall we track this in a separate PR?